data/templates/markdown/namespace.md.twig

```diff
@@ -1,4 +1,7 @@
 {% extends './layout.md.twig' %}
 {% block content %}
 {{ block('table_of_contents') -}}
+{% if ( node.children|length == 0 ) and ( node.traits|length == 0 ) and ( node.interfaces|length == 0 ) and ( node.classes|length == 0 ) and ( node.constants|length == 0 ) and ( node.functions|length == 0 ) %}
```

Comment:
If we do something like:

```
{% if ( node.children|length != 0 ) or ( node.traits|length != 0 ) or ( node.interfaces|length != 0 ) or ( node.classes|length != 0 ) or ( node.constants|length != 0 ) or ( node.functions|length != 0 )  or ( node.methods|length != 0 )%}
# {% block title %}{{ macros.buildNamespaceBreadcrumb(node)|spaceless|replace({"\n": "", "\r": "", "\t": ""}) }}{% endblock %}

{% block content %}{% endblock %}
{% block footer %}{% endblock %}
{% endif %}
```

in `layout.html.twig`. then in `html-to-md.php` we do something like:

```
    $content = file_get_contents($file);

	if (empty($content)) {
		unlink( $file);
		echo 'DELETED' . PHP_EOL;
		continue;
	}
```

I believe we won't need to have a specific string to test for in the script and we can just check if the file is empty or not.

Reply:
I made it slightly different, but the general idea was the same - check for an empty file instead of a specific string (much better).
I had to do it this different way because otherwise, we'd still have to check for a `# ` string - this would be the file's content when empty.

I also like it a little better because we can replace the heading - if we want to.